frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/PipelineParameters.tsx

```diff
@@ -20,6 +20,7 @@ const PipelineParameters: React.FC<PipelineParametersProps> = (props) => {
       emptyValues={{ name: '', description: '', default: '' }}
       emptyMessage={emptyMessage}
       isReadOnly={isReadOnly}
+      span={[4, 4, 4]}
```

Comment:
Shouldn't be required and therefore columns should be evenly spaced.
Same comment for all instances where columns should be evenly spaced.

Reply:
Not that we will be creating such layout but simply make them all even then add 1 to each column starting from the beginning until you've used up all the spaces.
eg.
5 = [3,3,2,2,2]
7 = [2,2,2,2,2,1,1]
9 = [2,2,2,1,1,1,1,1,1]